spec/presenters/supergroups/news_and_communications_spec.rb

```diff
@@ -1,9 +1,9 @@
+DEFAULT_IMAGE_URL = "https://assets.publishing.service.gov.uk/media/5e59279b86650c53b2cefbfe/placeholder.jpg".freeze
```

Comment:
Rather than being a global constant this should just be a variable. I think it could be localised to just the particular test it's used in as it only appears to be one, but otherwise could be a let.